firestore.rules

```diff
@@ -216,6 +216,32 @@ service cloud.firestore {
       allow delete: if isSuperAdmin() || isMemberOfParent(document, 'kpis') || isAdminOfParent(document, 'kpis');
     }
 
+    match /apiClients/{document} {
+      function isAdminOfQueriedParent() {
+        let userDoc = getUserDoc();
+        let userIsAdmin = isAdmin();
+        let parentDoc = getAfter(resource.data.parent);
+        let hasOrgDocumentInParent = 'organization' in parentDoc.data;
+        let isAdminFromOrgOfProdOrDep = userIsAdmin && hasOrgDocumentInParent && getAfter(parentDoc.data.organization).id in userDoc.data.admin;
+        let isAdminOfParent = userIsAdmin && !isAdminFromOrgOfProdOrDep && parentDoc.data.id in userDoc.data.admin;
+        return isSignedIn() && (isAdminFromOrgOfProdOrDep || isAdminOfParent);
+      }
+      function isMemberOfQueriedParent() {
+        let userRef = /databases/$(database)/documents/users/$(request.auth.token.email);
+        let parentDoc = getAfter(resource.data.parent);
+        let userIsMemberOfParent = userRef in parentDoc.data.team;
+        return isSignedIn() && userIsMemberOfParent;
+      }
+
+      allow read: if isSuperAdmin() || isMemberOfQueriedParent() || isAdminOfQueriedParent();
```

Comment:
Hvorfor kan ikke denne også bruke `isMemberOfParent`/`isAdminOfParent`...?

Reply:
Eksisterende `isMemberOfParent`/`isAdminOfParent` fungerer ikke ut av boksen, noe sted, for `read` (egentlig `list`tror jeg). Får se på den.